File: Sorts/selectionSort.go
package main

import "fmt"

func selectionsort(A []int) {
	for i := 0; i < len(A)-1; i++ {
		position := i
		for j := i + 1; j < len(A); j++ {
			if A[position] > A[j] {
				position = j
			}
		}
		temp := A[i]
		A[i] = A[position]
		A[position] = temp
	}
}

func main() {
	A := []int{3, 5, 8, 9, 6, 2}
	fmt.Println("Original Array: ", A)
	selectionsort(A)
	fmt.Println("Array after sorting: ", A)
}

File: Python/MISK/selectionSort.py
A = [8,2,6,5,1]
n = len(A)
for i in range (n-1):
    position = i
    for j in range (i+1,n):
        if A[j] < A[position]:
           position = j
        A[i],A[position] = A[position],A[i]
        # temp = A[i]
        # A[i] = A[position]
        # A[position] = temp
    print(A)